source/retryable-writes/tests/unified/aggregate-out-merge.yml

```
@@ -30,6 +30,8 @@ initialData:
   
 tests:
   - description: "aggregate with $out does not set txnNumber"
+    runOnRequirements:
+      serverless: forbid # $out is not supported on serverless
```

The linter is complaining about this:

```suggestion
      - serverless: forbid # $out is not supported on serverless
```